tests.py

```diff
@@ -7,14 +7,24 @@
 
 import os
 import unittest
+import threading
 from subprocess import Popen, PIPE
+from ipc.loader import load_plugin
 
 
 RULES_PATH = os.path.abspath(os.path.join('.', 'sample_rules'))
 
+def send(signal, value):
+    """Method overriden by the IPC plugin"""
+    pass
 
 class TestVSM(unittest.TestCase):
-    def run_vsm(self, name, input_data, expected_output, use_initial=True):
+
+    def setUp(self):
+        load_plugin('ipc.zeromq', use_receive=False)
```

Comment:
I don't know if code reuse here is that important. Maybe you should just have zeromq code on the test that can send the messages? It's only a few lines.